topics/kmm-plugin-releases.md

```diff
@@ -34,12 +34,39 @@ Release info
 Release highlights
 </th>
 <th>
-Compatible Kotlin plugin version
+Compatible Kotlin version
 </th>
 </tr>
 <tr>
 <td>
 
+**0.2.1**
+
+Released: February 15, 2021
+
+</td>
+<td>
+
+* [Support for Kotlin 1.4.30](https://youtrack.jetbrains.com/issue/KT-44038).
+* [New compatibility policy – for each major version of Kotlin there will be a new version of KMM plugin](https://youtrack.jetbrains.com/issue/KT-43211). (TODO TBD remove this item?)
+* [Ability to launch common tests for Android on a local JVM via run gutter](https://youtrack.jetbrains.com/issue/KT-42463).
+* [Provided class names for variables in the variable view in the Kotlin/Native debugger](https://youtrack.jetbrains.com/issue/KT-34630).
```

Comment:
`provided` doesn't seem to a good word here. Ability to view class names. 
Is `the variable view` important here?  Or is it clear from other context?

Reply:
>provided
I tried to find a synonym, thanks for your variant.

>the variable view
Seems like this part may be thrown, thank you.